Makefile

```diff
@@ -596,20 +596,24 @@ image-to-cluster: image openscap-image namespace openshift-user  ## Builds and p
 	$(eval IMG = image-registry.openshift-image-registry.svc:5000/openshift/$(APP_NAME):$(TAG))
 	$(eval OPENSCAP_IMAGE = image-registry.openshift-image-registry.svc:5000/openshift/$(OPENSCAP_NAME):$(OPENSCAP_TAG))
 
-.PHONY: e2e-content-images
-e2e-content-images:  ## Build the e2e-content-image
+.PHONY: build-e2e-content-images
```

Comment:
I think renamed these targets from build-* because "make build-..." sounded redundant in my head :)
But these are primarily dependencies, so its no big deal.